trident/absorption_spectrum/absorption_spectrum_fit.py

```diff
@@ -248,7 +253,13 @@ def _complex_fit(x, yDat, yFit, initz, minSize, errBound, speciesDict,
         dictionary containing all relevant parameters needed
         to create an absorption line of a given species (f,Gamma,lambda0)
         as well as max and min values for parameters to be fit
+    
+    :maxNumComps: int
 
+        maximum number of voight profiles to fit at once. If the loop is
```

Comment:
```suggestion
        maximum number of voigt profiles to fit at once. If the loop is
```

Reply:
It's crazy that I still get this wrong so often. Thanks for catching it!